teemaderegister-be/src/controllers/auth.js

```diff
@@ -8,6 +8,37 @@ const { signToken, blacklistToken } = require('../utils/jwt')
 
 const { Error } = require('../utils/errors')
 
+
+module.exports.emailVerification = async (req, res) => {
+  //const {email, password} = req.body
+  //return res.status(201).send({ message: req.params.token })
+
+  const user = await User.findById(req.query.id)
```

Comment:
frontend should call this endpoint if emailConfirmToken is in the url. Then user should be taken from req.user.id

and endpoint should provide response 200 if everything is OK. Frontend should render toast message notification that email verified.